Documentation/Signoffs/jdjoyce42-power-signoff.md

```diff
@@ -0,0 +1,188 @@
+
+
+
+
+
+
+## Buildable schematic 
+
+![image](https://github.com/CarsonDPope/Active-Noise-Control-With-Wall-Transmission-Detection/assets/123997954/b080a62f-e3b4-448b-8399-2f3279b2c822)
+
+
+*Figure 1. Power Subsystem buildable schematic.*
+
+The Error and Input will be wired by using jumper wires. 
+The Main processor will be wired by using a USB 2.0 micro.
+
+
+## Analysis
+| DEVICE            | Maximum Required Voltage | Maximum Required Current |
+| ----------------- | ------------------------ | ------------------------ | 
+| Main Processor    | 5 VDC                      | 100 mA                   | 
+| Input             | 5 VDC                      | 8.5 mA                   | 
+| Error             | 5 VDC                     | 8.5 mA                   | 
+| Output            | 12 VDC                    | 1.33 A                   | 
+
+The above table describes the maximum required supply voltage and supply current for each subsystem. The input and error subsystems require an identical amount of power to function. There are devices that need power in the input and error subsystems respectively. 
+
+Main Processor
+The main processor is rated to run at 5 VDC and 500 mA, this means that the total power consumed would be 
+~~~math
+P_{Main Processor} = VI = (5V)(500mA) = 2.5W
+~~~
+
+Input and Error
+- The TS472 microphone preamplifier must be driven with a 5 VDC signal at a maximum supply current of 2.4 mA [5].
+- The LM741 operational amplifier will be driven with a +5 VDC signal and a -5 VDC signal at a maximum supply current of 2.8 mA [6].
+- The CMA-4544PF-W electret microphone will be driven with a 5 VDC signal at a maximum supply current of 0.5 mA [7]. 
+
+Therefore, the total supply current will be 2.4 mA + 2(2.8 mA) + 0.5 mA = 8.5 mA. Since the LM741 requires two supply pins, the maximum supply current must be doubled. The total output power for both input and error combined is:
+~~~math
+P_{input/output} =2(VI) = 2(5 V * 8.5 mA) = 2(42.5 mW) = 85 mW
+~~~
+
+Output
+
+The output subsystem consists of an audio amplifier (AA-AB32231) and a speaker. According to the data sheet for the AA-AB32231 [9], the audio amplifier will operate on 12 VDC and 1.3 A, so the output power is:
+~~~math
+P_{Output} = VI =(12V)(1.33A) = 15.96W
+~~~
+
+
+Total Power
+
+Adding all the power draws together: 
+~~~math
+P_{Total} = 15.96 W + 42.5 mW + 42.5 mW + 2.5 W = 18.545 W
+~~~
+
+The efficiency of the buck converter is 92% so the power loss between input and outputs will be about 8% percent. The total output power required by the systems the buck converter will feed is 2.585 W. Multiplying this by 8% equals 0.2068 W.
```

Comment:
Your math is incorrect. This is not how to find the current actually required assuming 8% loss.